lib/logger.js

```diff
@@ -67,11 +67,14 @@ const streamConverter = {
         };
     },
     syslog(stream, conf) {
-        const impl = syslogStream.createBunyanStream({
-            facility: stream.facility || 'local0',
-            host: stream.host || '127.0.0.1',
-            port: stream.port || 514
-        });
+        const defaultOpts = {
+            facility: 'local0',
+            host: '127.0.0.1',
+            port: 514,
+            prefix: ''
+        };
+
+        const impl = syslogStream.createBunyanStream(Object.assign(stream, defaultOpts));
```

Comment:
This should be other way around. The right argument overrides the left argument.